packages/sveltekit/package.json

```diff
@@ -53,7 +53,7 @@
     "@babel/types": "7.20.7",
```

Comment:
this seems to fail when building now. I tried locally, and just removing the `@babel/types` dependency seems to "fix" this - can you try adjust it like this?

Reply:
Super weird! Things build locally for me correctly. Perhaps this is Yarn v1 not correctly layout things out on disk? I've run into that before. Let me start from a fresh repo and see if I can reproduce and fix.

Reply:
I unpinned `@babel/types` in 40bbc53 to allow a newer version to be used.

The build succeeded after the following:

```sh
rm -rf packages/*/node_modules
rm -rf node_modules
rm -rf .nxcache
yarn
yarn build
```

Let's see if CI is happy with this.